SPECS/reaper/reaper_build_caches.sh

```diff
@@ -0,0 +1,168 @@
+#! /bin/bash
+
+set -euxo pipefail
```

Comment:
I'd also remove the `-x` from the final script. It's useful for debugging but otherwise generates a ton of noise.
```suggestion
set -euo pipefail
```

Reply:
Done